### DISCUSSION TITLE
Client Side JS library

### DISCUSSION THREAD

**Discussion by C4RR13P0TT3R** [jgraph/drawio#3011] [Q&A]
Hello Guys,

I am a frequent user of your software as a consumer of the desktop app and integrations in JIRA etc..

However as part of an own software project I would like to integrate a tool with your or similar capabilities into a React JS client. There's some suitable libraries for drawing and graph editing available on the market for license fees far from my budget. Furthermore they have a sufficient but not perfect fit. If I got it right you have maintained mxgraph as a suitable library until some years ago. Is there a possibility to use maintained software as part of my project or is it suitable to use mxgraph?

Best regards

**Comment by davidjgraph** [comment]
I don't understand the question.

**Reply by C4RR13P0TT3R** [reply]
Is it possible to integrate drawio into my React client?

**Reply by davidjgraph** [reply]
OK, thanks for the clarification. I don't know, I've never used React.
